docs/concepts/wg.md

```diff
@@ -1,37 +1,37 @@
+ - For participants, the principle of "can I contribute something essential to the WG" applies. Pure interest in the topic is no legitimation to participate in a WG.
+ - Each WG participant brings the required time.
 
-## Aufbauorganisation
- - Eine Arbeitsgruppe ist einer Leadorganisation (CC, Leitung, etc.) zugeordnet.
- - Die Arbeitsgruppe hat
-   - Eine Lead Organisation (z.B. ein CC)
-   - einen Arbeitsgruppen-Lead (eine Person)
-   - 2-10 zusätzliche Teilnehmer, idealerweise aus 2 oder mehr Domänen und wenn sinnvoll aus mehreren Leadgremien
-   - ein konkretes, formuliertes Ziel
-   - einen geplanten Endtermin
- - Der Arbeitsgruppen-Lead entscheidet abschliessend über die Zusammensetzung der Arbeitsgruppe.
+## Organizational structure
+ - A working group is assigned to a lead organization (CoP, enterprise architecture, management, etc.).
+ - The working group has
+   - A lead organization (e.g. a CoP)
+   - A working group lead (one person)
+   - 2-10 additional participants, ideally from 2 or more domains and if appropriate from multiple other teams or CoPs
+   - A specific, articulated goal
+   - a planned start and end date
+ - The working group lead makes the final decision on the composition of the working group.
 
-## Ablauforganisation
- - Jede Arbeitsgruppe führt eine Confluence-Page auf der sowohl Lead, Ziel/Auftrag, Termine,  Teilnehmer, (Zwischen-)Ergebnisse, Entscheidungen und Bewertungen des Leadgremiums protokolliert werden. 
-- Arbeitsgruppen haben die Status **geplant** (mit Starttermin), **aktiv** (mit Endtermin) und **geschlossen**. Der Status ist auf der Confluence-Page ersichtlich.
-- 1-2 Wochen vor dem Start einer Arbeitsgruppe, wird diese im nächsten [Leitungsprotokoll](https://confluence.baloisenet.com/atlassian/x/WYHbLw) publiziert. Interessenten (auch nicht TAM-Mitglieder) können sich dann beim Lead melden. 
-- Grundsätzlich gilt bei Entscheidungen das Mehrheitsprinzip (absolutes Mehr) wobei der Lead das Recht des Stichentscheids hat. In begründeten Fällen kann von diesem Modus abgewichen werden.
-- Besteht das Bedürfnis zur Eskalation ist der CCO der erste Ansprechpartner.
-- Am Ende einer Arbeitsgruppe wird das Resultat dem Leadgremium vorgestellt und von diesem bewertet.
-- Resultat und Bewertung des Leadgremiums werden im folgenden Leitungsprotokoll publiziert.
+## Process organization
+ - Each working group maintains a Confluence page on which the lead, goal / task, dates, participants, (interim) results, decisions and evaluations of the lead committee are logged. 
+- Workgroups have the status **planned** (with start date), **active** (with end date) and **closed**. The status can be seen on the Confluence page.
+- 1-2 weeks before the start of a working group, it will be published in the next [IT management protocol](https://confluence.baloisenet.com/atlassian/x/WYHbLw). Interested parties can then contact the lead. 
```

Comment:
Do we want to keep confluence links in public GitHub repos?

Reply:
Good question: (imho) due to the fact that we publish confluence, bitbucket and JIRA plugins as open-source for years now it's not a secret using these tools. In addition to that using the unique permalink version does not expose any additional information other than that. Due to that I'd say : yes.